### PR TITLE
add missing device types

### PR DESCRIPTION
**Pull request by epwalsh**
This just add a missing device type `TV`

### CODE REVIEW

**Comment by ramsayleung** [comment]
From the Doc about [Device types](https://developer.spotify.com/documentation/web-api/reference/player/get-a-users-available-devices/#device-types), it seems more devices type which I simply ignore should be added to the `DeviceType` enum

File: src/spotify/senum.rs [hunk 1]
```diff
@@ -1219,5 +1219,6 @@ pub enum DeviceType {
     Computer,
     Smartphone,
     Speaker,
+    TV,
```
Comment (ramsayleung) [inline src/spotify/senum.rs]: Perhaps You should update the related Doc about Device type in case of confusing other developers.

Comment (epwalsh) [reply src/spotify/senum.rs]: Good point. I just added the remaining device types and put a link in the docs.

**Comment by epwalsh** [comment]
@ramsayleung could we bump the version and publish to crates with this patch? Need this patch to resolve a bug on spotify-tui: https://github.com/Rigellute/spotify-tui/issues/39

**Comment by ramsayleung** [comment]
> @ramsayleung could we bump the version and publish to crates with this patch? Need this patch to resolve a bug on spotify-tui: [Rigellute/spotify-tui#39](https://github.com/Rigellute/spotify-tui/issues/39)

Of course we could, I will bump the version to 0.7.0 now :)

**Comment by ramsayleung** [comment]
Well, there we are, rspotify is [0.7.0](https://crates.io/crates/rspotify)

File: src/spotify/senum.rs (hunk 1 above)
Comment (ramsayleung) [reply src/spotify/senum.rs]: Cool

**Comment by epwalsh** [comment]
Thanks @ramsayleung!

**Pull request merged**